client/src/pages/Reviews/Reviews.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ReviewsStyled from "./ReviewsStyled";
+import  { Review } from "./Review"
+
+const Reviews = () => {
```

Comment:
I would put the map for each Review here in the Reviews file so it would map the array and create a Review element for each review in the array, instead of the Reviews page just returning the map, does that make sense?